### PR TITLE
Import new tmRNA data

### PR DESCRIPTION
**Pull request by blakesweeney**
This adds a parser for tmRNA data along with some tests and docs. This should be ready to go, but could you use a quick sanity check. I've run a simple test on parsing the current data file we have and that doesn't fail.

### CODE REVIEW

File: rnacentral_pipeline/cli/__init__.py
Comment (afg1) [inline rnacentral_pipeline/cli/__init__.py:1]: This is the start of the cli refactor I guess? I think it makes sense

File: workflows/parse-databases.nf [hunk 1]
```diff
@@ -24,15 +24,16 @@ include { quickgo } from './databases/quickgo'
...
 include { ribovision } from './databases/ribovision'
 include { sgd } from './databases/sgd'
 include { silva } from './databases/silva'
 include { snodb } from './databases/snodb'
 include { snorna_database } from './databases/snorna_database'
 include { tarbase } from './databases/tarbase'
+include { tmrna } from './databases/tmrna'
 include { zfin } from './databases/zfin'
 include { zwd } from './databases/zwd'
+include {ribocentre } from './databases/ribocentre'
```
Comment (afg1) [inline workflows/parse-databases.nf:36]: Is there a reason this is now at the bottom?

**Review by afg1 — APPROVED** [review]
I think this all looks fine from a quick look. Only one very pedantic thing - ribocentre has jumped out of order in `workflows/parse-databases.nf` but I don't know if there is some higher logic being applied.

**Comment by blakesweeney** [comment]
I sorted the imports, which is what led to that moving. Pretty silly and I will avoid that in the future.

**Pull request merged**